Create rain.thedev.me.json

GodTierCutie opened the pull request:
(proud to be the first one heh)
but Idk about DNS records that much, so I wonder if this is right, hmm

YazeedT commented:
Your JSON is invalid.

YazeedT commented:
Your description is empty. Please add one.

YazeedT reviewed (approved):
Checks passed. All good. Enjoy your new subdomain.

GodTierCutie committed:
Create rain.thedev.me.json

(proud to be the first one heh)
but Idk about DNS records that much, so I wonder if this is right, hmm

YazeedT commented:
Hello Rain. Thank you for sending your pull request.

Neocities requires you to be a supporter to have custom domains. Then follow the instructions Neocities has on their connect a custom domain page.

Moreover, your JSON lists all possible records, but you must only keep and fill in the ones you want. In your case, it would be either A record, or CNAME record pointing at your Neocities blog, depending on their settings and options.

Resend your pull request after reviewing everything and making sure it is setup correctly.
